libdrgn/python/program.c

```diff
@@ -734,6 +734,66 @@ static StackTrace *Program_stack_trace(Program *self, PyObject *args,
 	return ret;
 }
 
+static PyObject *Program_symbols(Program *self, PyObject *args, PyObject *kwargs)
+{
+	static char *keywords[] = {
+		"name", NULL,
+	};
+	struct drgn_error *err;
+	struct drgn_symbol **symbols;
+	size_t count;
+	PyObject *list, *arg = Py_None;
+
+	if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|O", keywords, &arg))
```

Comment:
You should include the method name here so that argument parsing exceptions include the method name:

```suggestion
	if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|O:symbols", keywords, &arg))
```

(This still applies to `PyArg_ParseTuple`.)